Jenkinsfile

```diff
@@ -144,11 +150,20 @@ pipeline {
           // perform any needed modifiers on the build parameters here
           sbtbuild.setRunITTest(true)
           sbtbuild.setNoSubproject(true)
-          sbtbuild.setScalaVersion("2.12")
+          sbtbuild.setScalaVersion(scala_version)
 
           // build
           echo "Building sbt project..."
           sbtbuild.build()
+
+          // If we're cutting, also publish libraries if they haven't already been published for this version
+          if(stage_cut == true) {
+            result = sh(returnStdout: true, script: "#!/bin/sh -e\ncurl -s -o /dev/null -w \"%{http_code}\" -u ${ARTIFACTORY_USER}:${ARTIFACTORY_PASSWORD} https://repo.socrata.com/artifactory/libs-release/com/socrata/coordinatorlib_${scala_version}/${sbtbuild.getServiceVersion()}/coordinatorlib_${scala_version}-${sbtbuild.getServiceVersion()}.pom").trim()
+            echo "Check for published artifact result: [${result}]"
+            if(result != "200") {
```

Comment:
if this is expecting 404, use result == "404".

Reply:
Done!